src/jose/IsomorphicJoseUtility.ts

```diff
@@ -56,10 +56,13 @@ export default class IsomorphicJoseUtility implements IJoseUtility {
     options?: JoseJWT.SignOptions
   ): Promise<string> {
     const parsedKey = await JWK.asKey(key);
+    console.log(1);
     const convertedKey: string = parsedKey.toPEM(true);
+    console.log(2);
```

Comment:
Debug code left behind?

Reply:
Fixed in 3b125fb833c36085d795ab1cd1a732e7ce21b50a